sql/item_mods_pet.sql

```diff
@@ -114,7 +114,7 @@ INSERT INTO `item_mods_pet` VALUES (13975,27,-2,1);     -- Evoker's Bracers Avat
 INSERT INTO `item_mods_pet` VALUES (14103,27,-2,1);     -- Evoker's Pigaches Avatar: Enmity -2
 INSERT INTO `item_mods_pet` VALUES (14103,68,5,1);      -- Avatar: Enhances evasion +5 (?)
 INSERT INTO `item_mods_pet` VALUES (14227,3,10,2);      -- Drachen Brais Wyvern: HP+10%
-INSERT INTO `item_mods_pet` VALUES (14228,25,10,1);     -- Evoker's Spats Avatar: Enhances accuracy +10 (?)
+INSERT INTO `item_mods_pet` VALUES (14228,25,14,1);     -- Evoker's Spats Avatar: Enhances accuracy +14
```

Comment:
Where'd you pull this stat from?

afaik it should be +10 and was checking in retail using /checkparam

Reply:
https://kegsay.livejournal.com/4471.html

Specifically lists the item and the value. I can revert it if it is confirmed that this has since changed.

Reply:
Please be careful with the age of your information.

Newer sources claim +10 accuracy using /checkparams on retail.

Reply:
I believe it but I couldn't find them. Spent quite a bit of time checking for attack, accuracy, and evasion both in en and jp sources.

Reply:
Both BGWiki and FFXIclopedia state +10 state with /checkparam, which is much more accurate than work that is essentially guessing by comparison.

Reply:
You're right. I was checking for https://ffxiclopedia.fandom.com/wiki/Summoner's_Bracers, which has no info. Went looking for it, found kegsay, and he mentioned spats as well so I changed it as well. Thx.